tests/unit/resources/servers/test_server_hardware.py

```diff
@@ -86,6 +86,17 @@ def test_add_called_once(self, mock_create):
         self._server_hardware.add(information)
         mock_create.assert_called_once_with(information.copy(), timeout=-1)
 
+    @mock.patch.object(ResourceClient, 'create')
+    def test_add_multiple_called_once(self, mock_create):
+        information = {
+            "licensingIntent": "OneView",
+            "configurationState": "Managed"
+        }
+        mock_create.return_value = {}
+
+        self._server_hardware.add_multiple(information)
+        mock_create.assert_called_once_with(information.copy(), timeout=-1, uri="/rest/server-hardware/discovery")
```

Comment:
declare a uri variable and give the variable name here.
(to match with the coding style of this module and reduce the line length)